src/components/modals/export-map-modal.js

```diff
@@ -58,14 +58,17 @@ const StyledInput = styled.input`
   }
 `;
 
+const DISCLAIMER = "* We will provide a temporary token. If you don't provide your own token, " +
+  "map will break. You can later follow this instruction to edit html and input token.";
```

Comment:
> If you don't provide your own token, map will break
If you don't provide your own token, the map may fail to display at any time as we rotate our token to avoid misuse of the token. 

If you don't provide your own token, the map may fail to display at any time as we rotate our token to avoid misuse of the token. 

> You can later follow this instruction 

You can change the Mapbox token at any time using the following instructions:

Reply:
can just say `If you don't provide your own token, the map may fail to display at any time as we rotate ours to avoid misuse`.  shorter and less use of the word `token`